example-app/SantokuApp/src/bases/logging/Logger.ts

```diff
@@ -94,14 +94,13 @@ class Logger {
 
   /**
    * errorレベルのログを出力します。
-   * @param message ログメッセージ
+   * @param error エラー
    * @param errorCode エラーコード
    * @returns ロガーインスタンス
    */
-  error(message: string | LogMessageSupplier, errorCode: string): Logger {
+  error(error: Error, errorCode: string = 'UnexpectedError'): Logger {
```

Comment:
With string, the stack trace cannot be checked by logging products such as Clashlytics, so it is limited to Error.

Also, the developer is basically assuming that Logger.error is not called directly, but rather handleError is called.

Since the argument of handleError is set to unknown, it is also possible to pass a string. (internally wrapped in Error and passed to Logger.error)

So, I think that Logger.error can receive only Error, but what do you think?

Reply:
understood.